File: 509/Solution.py
class Solution:
    def fib(self, n: int) -> int:
        i,j,sum = 0,1,0
        if n==0:
            return 0
        if n==1:
            return 1
        
        for x in range(2,n+1):
            sum = i+j
            i = j
            j = sum
        
        return j
    
	

File: 509/Solution.java
class Solution {
    public int fib(int n) {
        int i=0,j=1,sum=0;
        if (n==0 || n==1){
            return n;
        }
        for(int x=2;x<n+1;x++){
            sum = i+j;
            i = j;
            j = sum;
        }
        
        return j;
    }
}